### ISSUE TITLE
Running LoRa Node and Nono-Gateway Node on the same machine

### ISSUE THREAD

**Issue by secirugy** [pycom/pycom-libraries#104]
I have 2 COM ports that I set on my machine: 1 for PySence board and 1 for PyTrack board.
Although I could use simple Expansion Board I did not mind to spend more and having these board for future use. The PySence board is used with MQTT example (which I modified to my needs). It reads the data from MQTT Broker over WiFi and delivers it to LoRa socket. The PyTrack serves as Nano-Gateway: it uses example code from here:
[https://github.com/pycom/pycom-libraries/tree/master/examples/lorawan-nano-gateway](url)

I can run each project from a separate window in VSCode and they work fine...up to the point that I launch them both. Then one of them stops...because they both fighting for the same COM port despite there are two...

The bottom line is that I am looking for a way to configure VSCode/PyMakr/Node.js or whatever in order that each project will connect to it is own specified COM port, in my case COM15 and COM19.
